### PR TITLE
add status filter to superfeature viewset

### PR DESCRIPTION
**Pull request by MichaelButkovic**
@mparent61 @benghaziboy

### CODE REVIEW

File: bulbs/super_features/filters.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import django_filters
+from rest_framework import filters
+
+from bulbs.super_features.utils import get_superfeature_model
+
+SUPERFEATURE_MODEL = get_superfeature_model()
+
+
+def filter_status(queryset, value):
```
Comment (MichaelButkovic) [inline bulbs/super_features/filters.py]: note: this isn't my favorite thing ever, but status is currently a property (that is different site side vs. bulbs side), so this seemed like an ok way to do this, unless we want to move status to being a model field for `Content`

**Comment by mparent61** [comment]
LGTM (w/ chaining changes we discussed)

File: tests/super_features/test_viewsets.py [hunk 2]
```diff
@@ -86,3 +88,11 @@ def test_super_feature_listing_search(self):
 
         self.assertEqual(len(response.data['results']), 1)
         self.assertEqual(response.data['results'][0]['title'], self.parent.title)
+
+    def test_super_feature_listing_filtering(self):
+        import pdb; pdb.set_trace()
```
Comment (benghaziboy) [inline tests/super_features/test_viewsets.py]: ----

Comment (mparent61) [inline tests/super_features/test_viewsets.py]: Agreed

Comment (MichaelButkovic) [inline tests/super_features/test_viewsets.py]: image

**Pull request merged**